internal/grpc/logstreamer/server.go

```diff
@@ -14,7 +14,9 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
-var logContextKey = struct{}{}
```

Comment:
The end of an era :smiling_face_with_tear:

Reply:
yep! :/

Reply:
Glad that I did the right thing on UP4W :) I even had a JIRA card to double-check that with didrocks